README.md

```diff
@@ -40,7 +40,7 @@ This controller watches for changes on `Policies` in the cluster namespace on th
 ## Geting started
 
 Go to the
-[Contributing guide](https://github.com/open-cluster-management-io/community/blob/main/sig-policy/contribution-guidelines.md)
+[Contributing guide](https://github.com/stolostron/community/blob/main/sig-policy/contribution-guidelines.md)
```

Comment:
This doesn't need to be replaced

Reply:
Thanks. That was a find/replace error on my part.